### DISCUSSION TITLE
ipv6 not working

### DISCUSSION THREAD

**Discussion by vicn1222** [angristan/openvpn-install#1406] [Help]
WOW, I was probably the 1st person used this script a long time ago, and now this script becomes so popular with so many issues! :-)

Anyway, excellent job, thank you!

When I installed openvpn using this script, I turn on the ipv6 support, but when I connect to the vpn server, there is no ipv6.

I use https://test-ipv6.com for ipv6 test using iPhone. No ipv6 detected.

Can the author please check the script?

Keep up all the good work!

**Comment by pathia** [comment]
check if you are missing ip6tables nat entries using
# sudo ip6tables -t nat -L
You probably have none. Then check the following PR.
https://github.com/angristan/openvpn-install/pull/1140
It helped me out.

**Comment by angristan** [comment]
Hi! have you check that the server indeed has ipv6?
